src/assets/css/main.css

```diff
@@ -375,6 +375,10 @@ br.big {
   border-radius: 4px;
 }
 
+.default-font-size {
+  font-size: .9375rem !important;
+}
```

Comment:
What's this doing here?

Reply:
Not sure where else to put it? Its specifically for the text in the 'filter by mod/user' dropdown, which has a fontsize of '14px' from '.choices__inner' style.